content/en/docs/test.md

```diff
@@ -92,10 +92,10 @@ culpa qui officia deserunt mollit anim id est laborum.
 3.  For single-digit numbered lists, using two spaces after the period makes
     interior block-level content line up better along tab-stops.
 
+    <!-- separate lists -->
 
-
-1.  This is a new list. It has two blank lines from the previous list, If it only
-    had one blank line, it would still show up as part of the previous list.
+1.  This is a new list. With Hugo, you need to use a HTML comment to separate two
+      consecutive lists.
 2.  Numbered lists can have paragraphs or block elements within them.
 
     Just indent the content to be even with the text of the bullet point, rather
```

Comment:
All of this block-level content is rendered outside of the list, and then a new list starts at line 108. :(